File: ueueo-framework/ueueo-localization/src/main/java/Abp/Localization/ILocalizationDictionary.cs
using System.Collections.Generic;
using Microsoft.Extensions.Localization;

namespace Volo.Abp.Localization;

/**
 * Represents a dictionary that is used to find a localized string.
*/
public interface ILocalizationDictionary
{
    String CultureName;//  { get; }

    LocalizedString GetOrNull(String name);

    void Fill(Dictionary<String, LocalizedString> dictionary);
}

File: ueueo-framework/ueueo-localization/src/main/java/com/ueueo/localization/ILocalizationDictionary.java
package com.ueueo.localization;

import java.util.Map;

/**
 * Represents a dictionary that is used to find a localized string.
 */
public interface ILocalizationDictionary {
    String getCultureName();

    LocalizedString getOrNull(String name);

    void fill(Map<String, LocalizedString> dictionary);
}
